server/graphql/types/src/types/requisition.rs

```diff
@@ -213,6 +213,13 @@ impl RequisitionNode {
         Ok(InvoiceConnector::from_vec(result))
     }
 
+    pub async fn number_of_shipments(&self, ctx: &Context<'_>) -> Result<i32> {
```

Comment:
hmm, could also use the shipments loader, just select the ids in graphql and length in front end, it's not as direct but since we are using graphql already could re-use existing

Reply:
[7d73206](https://github.com/msupply-foundation/open-msupply/pull/4385/commits/7d73206b18caf1f8caf92a546317e77be0344ba6)

Reply:
https://github.com/msupply-foundation/open-msupply/compare/2667-Can-we-auto-confirm-requisition-after-the-shipment-has-been-created-from-the-requistion-...2667-Can-we-auto-confirm-requisition-after-the-shipment-has-been-created-from-the-requistion-(Work-around-types)